spec/lib/crawler/api/crawl_spec.rb

```diff
@@ -13,8 +13,15 @@
   let(:crawl_config) do
     Crawler::API::Config.new(
       domains: [
-        { url: }
-      ]
+        { url: url }
```

Comment:
```suggestion
        { url: }
```

Based on CI failure below. It's possible that the local env rubocop version is different and so this isn't autocorrecting properly locally. I suggest manually including this fix.

```
Inspecting 160 files
.........................................................................................................C......................................................
Offenses:
spec/lib/crawler/api/crawl_spec.rb:16:16: C: [Correctable] Style/HashSyntax: Omit the hash value.
        { url: url }
               ^^^

160 files inspected, 1 offense detected, 1 offense autocorrectable
```